### PR TITLE
Adding loop support  to playprompt operation. Adding media file details.

### PR DESCRIPTION
**Pull request by tituscha**
Adding loop support  to playprompt operation. Also adding some details of the media file type to be provided to playprompt action.

### CODE REVIEW

File: api-reference/beta/resources/mediainfo.md [hunk 1]
```diff
@@ -18,6 +18,22 @@ The media information used in actions for prompts.
 | resourceId     | String  | Unique identity of the resource. |
 | uri            | String  | Path to the resource.            |
 
+## Remarks
```
Comment (Lauragra) [inline api-reference/beta/resources/mediainfo.md]: Please remove this section - it is a non-standard H2 for a resource topic. Topics that don't follow the standard heading structure will fail validation in the future. @VinodRavichandran , is this content that we can cover in a concept topic in the Learn section?

Comment (tituscha) [reply api-reference/beta/resources/mediainfo.md]: Removed this section.

**Comment by Lauragra** [comment]
@tituscha , are you ready to merge/publish?

**Comment by tituscha** [comment]
> @tituscha , are you ready to merge/publish?

@Lauragra - Yes. Please merge/publish it.

**Pull request merged**